src/wallet/wallet.h

```diff
@@ -738,7 +738,10 @@ typedef std::variant<
     CKeyID,
     CScriptID,
     libzcash::SproutPaymentAddress,
+    libzcash::SproutViewingKey,
```

Comment:
I think having this as an available pattern is okay (since there's a 1:1 to spending keys), so e.g. if we decide to allow viewing keys as the source in `z_sendmany` then this would be fine to allow.

Reply:
Added https://github.com/zcash/zcash/issues/5523 to track this.